lib/services/flock.rb

```diff
@@ -1,12 +1,14 @@
 class Service::Flock < Service::Base
   title 'Flock'
 
-  string :url, :placeholder => 'Incoming Webhook URL',
-                 :label => 'Flock Incoming Webhook URL. <br />' \
-                   'To create an incoming webhook, go to your Flock admin panel and switch to "Webhooks" tab'
+  string :url, :placeholder => 'Flock Webhook URL',
+                 :label => 'Go to the <a href="https://apps.flock.co/crashlytics">Flock App Store</a> and install the Crashlytics app. <br />' \
```

Comment:
Hey @jladieu This is the url that we plan to have for the Crashlytics app in [Flock App Store](https://apps.flock.co) after it is moved to production, post the update of Flock service hook on Crashlytics. Currently, it could be found at [the staging version](https://apps.flock-staging.co/crashlytics) for your verification. I have also bumped up an email thread regarding that I started a while ago. Happy to discuss further.